pkg/utils/activity/event.go

```diff
@@ -106,6 +112,19 @@ func (n Network) String() string {
 	}
 }
 
+func ParseNetwork(s string) Network {
+	switch s {
```

Comment:
Probably should do a `strings.ToLower` and maybe a `strings.TrimeSpace` to `s` as well.